cmd/svcat/testdata/output/get-plans-by-class.txt

```diff
@@ -1,4 +1,7 @@
-   NAME             CLASS                 DESCRIPTION        
-+---------+-----------------------+-------------------------+
-  default   user-provided-service   Sample plan description  
-  premium   user-provided-service   Premium plan             
+   NAME              CLASS                      DESCRIPTION               BROKER    
++---------+--------------------------+--------------------------------+------------+
+  default   user-provided-service      Sample plan description          ups-broker  
+  premium   user-provided-service      Premium plan                     ups-broker  
+  default   another-provided-service   Another sample plan              ups-broker  
```

Comment:
These last two plans should not be in the output.

Reply:
Agreed, I think a bug has been introduced for this use case `svcat get plans --class CLASS_EXTERNAL_NAME`